src/ucl_util.c

```diff
@@ -1123,6 +1141,11 @@ ucl_include_common (const unsigned char *data, size_t len,
 					allow_url =  ucl_object_toboolean (param);
 				}
 			}
+			else if (param->type == UCL_ARRAY) {
+				if (strcmp (param->key, "path") == 0) {
```

Comment:
None of the other cases in this if ladder use keylen, should they all be changed to strncmp?